samples/word/50-document/manage-annotations.yaml

```diff
@@ -0,0 +1,344 @@
+order: 14
+id: word-document-manage-annotations
+name: Manage annotations
+description: Shows how to leverage the Writing Assistance API to manage annotations and use annotation events.
+author: OfficeDev
```

Comment:
Do we need to add this attribute to our other snippets as well?

Reply:
Hmm, not sure. Let's put it in the agenda for our next meeting.